src/org/exist/util/serializer/XMLWriter.java

```diff
@@ -373,47 +373,72 @@ public void comment(final CharSequence data) throws TransformerException {
         }
     }
 
-    public void cdataSection(final char[] ch, final int start, final int len) throws TransformerException {
+    public void startCdataSection() throws TransformerException {
```

Comment:
should/could be private or package protected when required for unit tests

Reply:
Nope it should be public.